GNUmakefile

```diff
@@ -1,4 +1,4 @@
-AMREX_HOME  ?= ../amrex
+AMREX_HOME  = ../amrex
```

Comment:
For the purposes of this PR I wouldn't include this change. BTW on your machine, if AMREX_HOME isn't define, it will use ../amrex anyway (hence the question mark)